src/modules/media/queries/get-object-identifier-tuple.graphql

```diff
@@ -0,0 +1,6 @@
+query getObjectIdentifierTuple($meemooIdentifiers: [String!]) {
+	object_ie(where: { meemoo_identifier: { _in: $meemooIdentifiers } }) {
```

Comment:
Optional remark: technically we don't need the count, we only need to know if there are related items

so technically this would be enough:
```
@Get('related/has-related'): boolean
```

And this query could be made more performant:
```
object_ie(where: { meemoo_identifier: { _in: $meemooIdentifiers } }, limit: 1) {
```

But having this count api call might be helpful in the future when we need to show how many related items there are.
And the performance gain is probably negligible.

Reply:
To make sure this isn't always true we'd need to add an exclusion `_in` which would increase the amount of data going to-and-fro and introduce a weird edge case where two related items in the same paginated result will each have a count of 0 related items.

quick & dirty;
```
query MyQuery($_in: [String!], $_in1: [String!] = "") {
  object_ie(where: {meemoo_identifier: {_in: $_in}, _and: {_not: {schema_identifier: {_in: $_in1}}}}) {
    schema_identifier
    meemoo_identifier
  }
}
```
```
{"_in": ["k649p3zf8j", "rf5k943h2c"], "_in1": ["808d1e7077a1426b84aeef3c1da2f2b28c82f4bf1c634665bab00b3ce61008e537220edcea634da7a94749e863c0fd4b", "0efa42d7723f42fa8651c39a34bc184c7db1d2b7d0e24c01b9643eb1d03017bb8952810d0ada4893aa6824df11f023e6"]}
```

Reply:
let leave it with the count as is. Seems easier